### PR TITLE
Switching device registration log for newly register device and re-connecting devices

### PR DESCRIPTION
**Pull request by akoserwal**
Addressing: https://github.com/nexodus-io/nexodus/issues/1063

### CODE REVIEW

File: internal/nexodus/nexodus.go [hunk 1]
```diff
@@ -471,10 +471,13 @@ func (ax *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 	if err != nil {
 		return fmt.Errorf("join error %w", err)
 	}
-
+	deviceRegLogMsg := "Successfully registered device"
+	if modelsDevice.Revision != 0 && modelsDevice.Revision > 1 {
```
Comment (russellb) [inline internal/nexodus/nexodus.go:475]:
```suggestion
	if modelsDevice.Revision > 1 {
```

You can drop the first check. It's redundant with the second one.

**Review by russellb — CHANGES_REQUESTED** [review]
Thanks for looking at this!

**Review by chirino — CHANGES_REQUESTED** [review]
You can't use revision to detect if it's the first restoration of a device.

File: internal/nexodus/nexodus.go [hunk 2]
```diff
@@ -471,10 +471,13 @@ func (ax *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 	if err != nil {
 		return fmt.Errorf("join error %w", err)
 	}
-
+	deviceRegLogMsg := "Successfully registered device"
+	if modelsDevice.Revision > 1 {
+		deviceRegLogMsg = "Reconnected as device"
```
[the remark this answers is not in the document]
Comment (akoserwal) [reply internal/nexodus/nexodus.go:476]: @chirino is there any other field which I can use?

[the remark this answers is not in the document]
Comment (chirino) [reply internal/nexodus/nexodus.go:476]: see the implementation of `createOrUpdateDeviceOperation` you could have it return another value indicating if the device was created or it had existed.

**Review by chirino — APPROVED** [review]
LGTM

**Review by russellb — APPROVED** [review]

**Pull request merged**